openfifth/aspen-discovery
# Grapes js user testing updates

TEST PLAN:
1. Apply the patch. 
2. Set up Web Builder and navigate to the Grapes Page section. 
3. When creating a new grapes page, notice that there is now the option in the templates dropdown to have no template. 
4. Navigate to the Grapes Editor, the text for the labels for the grapes components should now be bigger and the save button for the editor should be highlighted in green.
